Formula/pymupdf.rb

```diff
@@ -0,0 +1,73 @@
+class Pymupdf < Formula
+  desc "Python bindings for the PDF toolkit and renderer MuPDF"
+  homepage "https://github.com/pymupdf/PyMuPDF"
+  url "https://files.pythonhosted.org/packages/19/2d/73cb79152442ace5a6f55de17755e7c4c0dbed5ac6180baa1767d6a0e279/PyMuPDF-1.20.0.tar.gz"
```

Comment:
Since this is packaged on PyPi, what's wrong with doing `pip install PyMuPDF`?

We typically don't package pure Python libraries unless they're used in many other formulae.